packages/run-it/src/RunIt.tsx

```diff
@@ -260,6 +268,18 @@ export const RunIt: FC<RunItProps> = ({
             <PerfTracker perf={perf} configurator={configurator} />
           </TabPanel>
         )}
+        {isExtension ? (
```

Comment:
Get rid of the ternary?
!isExtension instead?
The current true side is <></> which is nothing

Reply:
When I try to do that there are syntax errors for `Tab` and `TabPanels`. Leaving as is.